src/ozone/ozone.py

```diff
@@ -520,6 +521,75 @@ def get_specific_parameter(
 
         return result
 
+    def get_search_results_historical(self, city: str):
+        r = requests.get(f"https://search.waqi.info/nsearch/station/{city}")
+        res = r.json()
+
+        city_id, country_code, station_name, city_url, score = [], [], [], [], []
+
+        for candidate in res["results"]:
+            city_id.append(candidate["x"])
+            country_code.append(candidate["c"])
+            station_name.append(candidate["n"])
+            city_url.append(candidate["s"].get("u"))
+            score.append(candidate["score"])
+
+        return pandas.DataFrame(
+            {
+                "city_id": city_id,
+                "country_code": country_code,
+                "station_name": station_name,
+                "city_url": city_url,
+                "score": score,
+            }
+        ).sort_values(by=["score"], ascending=False)
+
+    @functools.lru_cache(maxsize=128)
+    def get_historical_data(
+        self, data_format: str = "df", *, city: str = None, city_id: int = None
+    ) -> pandas.DataFrame:
+        """Get historical air quality data for a city
+
+        Args:
+            data_format (str, None): File format or None.
```

Comment:
All other Ozone methods have the following line for this part of their docstring. I think it's best to keep it uniform here too:
```
data_format (str): File format for data. Defaults to 'df'.
                Choose from 'csv', 'json', 'xlsx'.
```

Reply:
Got it!